plugins/module_utils/common.py

```diff
@@ -437,8 +439,8 @@ def _custom_condition(resource):
                     if condition['reason']:
                         return match.reason == condition['reason']
                 return False
-            status = True if match.status == 'True' else False
-            if status == condition['status']:
+            if status = True if match.status == 'True' else False
```

Comment:
Maybe this is my python not being amazing here, but this line seems wrong to me...?

Reply:
Damn, not it's me messing my `git rebase` :-)

Reply:
Okay good... I was like "I'm no Python God, and I know there can be some weird inline stuff (coming from PHP I've seen it all)... but that just doesn't seem like it would do anything good."

Reply:
Well, I did 10 years of Perl-fu in a previous life :D.